python/GangaTest/__init__.py

```diff
@@ -5,6 +5,8 @@ def getEnvironment(c = None):
     return {}
 
 def loadPlugins( config = None ):
+    import Lib.GListApp
```

Comment:
I thought the general move was such that we're looking to get rid of ```GangaTest``` once we've moved all tests away from it so will the tests run without these changes?